contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -130,6 +134,8 @@ The first part of the token is the `token-id`.  The second part is the `token-se
 
 This new type of token is different from the current CSV token authenticator that is currently part of Kubernetes.  The CSV token authenticator requires an update on disk and a restart of the API server to update/delete tokens.  As we prove out this token mechanism we may wish to deprecate and eventually remove that mechanism.
```

Comment:
Normal auth tokens are defined in a CSV file on disk.  We are scoping this mechanism to just bootstrap as we are storing the tokens in etcd.  The general recommended approach is, for day to day usage, use TLS client certs or another more full featured auth provider.